akka-http-core/src/main/java/akka/http/javadsl/model/headers/CacheDirectives.java

```diff
@@ -19,7 +19,7 @@ public static CacheDirective MAX_STALE() {
         return new akka.http.scaladsl.model.headers.CacheDirectives.max$minusstale(OptionConverters.toScala(Optional.empty()));
     }
     public static CacheDirective MAX_STALE(long deltaSeconds) {
-        return new akka.http.scaladsl.model.headers.CacheDirectives.max$minusstale(OptionConverters.toScala(OptionalLong.of(deltaSeconds)));
+        return new akka.http.scaladsl.model.headers.CacheDirectives.max$minusstale(OptionConverters.toScala(OptionalLong.of(deltaSeconds)).map(delta -> (Object)delta));
```

Comment:
This seems kinda unnecessary, can't we just call `new Some<>(deltaSeconds)` and skip the whole conversion?